Lab3.ipynb

```diff
@@ -0,0 +1,405 @@
+    "print('Neg True:', badDiscard)\n",
+    "print('Neg False:', goodDiscard)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Problem 1.A.4: According to my truth table for a 5 sigma stastistical significance probability, we are keeping 2 bad data values and all the good data values. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 117,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Sigma value: 1\n",
+      "Pos True: 68171\n",
+      "Pos False: 0\n",
+      "Neg True: 8\n",
+      "Neg False: 31829\n",
+      "\n",
+      "Sigma value: 2\n",
+      "Pos True: 95451\n",
+      "Pos False: 1\n",
+      "Neg True: 7\n",
+      "Neg False: 4549\n",
+      "\n",
+      "Sigma value: 3\n",
+      "Pos True: 99717\n",
+      "Pos False: 1\n",
+      "Neg True: 7\n",
+      "Neg False: 283\n",
+      "\n"
+     ]
+    }
+   ],
+   "source": [
+    "# 1.B:\n",
+    "for i in range(1,4): \n",
+    "    goodKept = goodData[np.where((goodData< 12+i*.4) & (goodData> 12-i*.4))]\n",
+    "    goodDiscard = len(goodData) - len(goodKept)\n",
+    "    badData = np.asarray(badData)\n",
+    "    badKept = badData[np.where((badData>12-i*.4) & (badData<12+i*.4))]\n",
+    "    badDiscard = len(badData) - len(badKept)\n",
+    "    print('Sigma value:', i)\n",
+    "    print('Pos True:', len(goodKept))\n",
+    "    print('Pos False:', len(badKept))\n",
+    "    print('Neg True:', badDiscard)\n",
+    "    print('Neg False:', goodDiscard)\n",
+    "    print()\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Problem 1.B: Above I reran the code with different sigma values and as you can see the larger the threshhold the less likely you are to throw away good data. This is because more data will meet the demands of our threshold as the tolerancesa relax. And these omissions are predictable since we are dealing with a gaussian distribution. We know that within 1, 2, and 3 standard deviations the percentage of data inside that range is 68%, 95%, and 99.7% respectively."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Problem 1.C: Yes, there can be mistakes of commission. In our example with 5 sigma we had two bad data points. However, they are avoidable by making the statistical threshold smaller. However, this comes at a price described in the answer above. If we decrease the threshold too much then we omit more good data. So finding a good balance between accepting good data and rejecting bad data will yield the most optimal threshold. "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Problem 2.1: With a gaussian distribution having a mean of 0 and a standard deviation of 1, what is the distance a potential asteriod must travel in order to distinguish itself by at at least 5 sigma from the bacgkround?"
```

Comment:
Problem 2.1:
- typo: duplicate typing of the word "at" in the question
- the second half of the statistical question makes sense. But the first half I don't think is quite complete, since it doesn't talk about the 2 dimensional aspect--I connected this to the last question in HW 3 from this week, where we looked at what the pdf is of a 2D Gaussian is if we "measure its amplitude". When I did that problem in the HW, I found on Wikipedia it is a Chi distribution for k=2, which I also just found out during the lab is a Rayleigh with sigma=1. So I think you should add a piece to your statistical question mentioning the 2 dimensional distribution, and discuss in the explanations what the pdf is.
- also copying the plot from the lab instructions could help.